include/CL/opencl.hpp

```diff
@@ -8859,6 +8920,64 @@ typedef CL_API_ENTRY cl_int (CL_API_CALL *PFN_clEnqueueReleaseD3D10ObjectsKHR)(
     {
         return detail::errHandler(::clFinish(object_), __FINISH_ERR);
     }
+
+#ifdef cl_khr_semaphore
+    cl_int enqueueWaitSemaphores(
+        const vector<Semaphore> &sema_objects,
+        const vector<cl_semaphore_payload_khr> &sema_payloads = {},
+        const vector<Event>* events_wait_list = NULL,
+        Event *event = NULL) const
```

Comment:
In new code, do not use macro `NULL`.
```suggestion
        const vector<Event>* events_wait_list = nullptr,
        Event *event = nullptr) const
```

Reply:
Shouldn't we change it in the whole file? It looks inconsistent now: there are more than 600 occurrences of the `NULL` and about 40 of the `nullptr`. If so, do you think it's a good idea to do it in another task?

Reply:
Yes probably a good task. The inconsistency is not my main concern when I look at the file as an antique shop... :-)

Reply:
Done